apps/web/src/app/components/ui/status.tsx

```
@@ -0,0 +1,25 @@
+    return (
+        !close ?
+            <div className={"relative"}>
+                <div
+                    className={`${status == 'success' ? 'bg-success text-success-foreground' : 'bg-destructive/20'} p-4 mb-4 text-sm rounded-md font-normal border-2 border-destructive/30 text-destructive-foreground`}>
+                    {message}
+                </div>
+                <Button variant={"ghost"} size={"icon"} onClick={() => setClose(true)}
+                        className={"absolute top-0 right-0 rounded-full hover:bg-transparent text-destructive"}>
+                    <X/>
+                </Button>
+            </div> : <div></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)